testing/integration/agent_long_running_leak_test.go

```diff
@@ -233,7 +233,7 @@ func (runner *ExtendedRunner) CheckHealthAtStartup(ctx context.Context) {
 				}
 			}
 			runner.T().Logf("component state: %s", comp.Message)
-			if comp.State != int(cproto.State_HEALTHY) {
+			if comp.State != int(cproto.State_FAILED) {
```

Comment:
I think the condition should be inverted, right?
```suggestion
			if comp.State == int(cproto.State_FAILED) {
```

Reply:
Yes it should, thanks!